dtfs-central-api/src/services/changeStream/changeStreamApi.test.ts

```diff
@@ -1,10 +1,26 @@
 import { Binary, ChangeStreamInsertDocument, ObjectId } from 'mongodb';
 import axios from 'axios';
-import { postAuditDetails } from './changeStreamApi.ts';
+import { DeletionAuditLog } from '@ukef/dtfs2-common';
+import { postAuditDetails, postDeletionAuditDetails } from './changeStreamApi';
 
 jest.mock('axios', () => jest.fn(() => Promise.resolve('mockResponse')));
 
 describe('changeStreamApi', () => {
+  const originalEnv = process.env;
+  beforeEach(() => {
```

Comment:
technically the process.env could be done in a before all

Reply:
The it.each below modifies the process.env so this resets it in between